demos/No_Position_Experiment.ipynb

```diff
@@ -1271,6 +1271,9 @@
                 "losses = []\n",
                 "loss_labels = []\n",
                 "for hook_name in hook_list:\n",
+                "    if hook_name not in cache:\n",
+                "        continue\n",
+                "\n",
```

Comment:
I had to skip a few keys because `model.hook_dict` contains more entries than `cache`:
```python 
>>> set(model.hook_dict.keys()) - set(cache.keys())

{'blocks.0.attn.hook_result',
 'blocks.0.hook_k_input',
 'blocks.0.hook_q_input',
 'blocks.0.hook_v_input',
 'blocks.1.attn.hook_result',
 'blocks.1.hook_k_input',
 'blocks.1.hook_q_input',
 'blocks.1.hook_v_input'}
```

Let me know if this is an issue!